src/main/java/org/assertj/core/util/diff/Delta.java

```diff
@@ -136,6 +136,7 @@ int lineNumber() {
   }
 
   String formatLines(List<T> lines) {
+    //TODO see what we should use here
     return STANDARD_REPRESENTATION.format(lines, DEFAULT_START, DEFAULT_END, ELEMENT_SEPARATOR_WITH_NEWLINE, "   ");
```

Comment:
Thanks for the explanation, I'll see if it makes sense to move `toStringOf(lines)` in the interface, gut feeling is we should not but I'll think about it. 

I was registering a `Representation` in assertj-examples, it worked but we really need to have some doc on how to do this, I will update the javadoc for `Assertion.useStandardRepresentation()` as it does not necessarily switch to `StandardRepresentation` anymore.